src/views/adminPage.ejs

```diff
@@ -29,9 +34,9 @@
       <% }); %>
     </tbody>
   </table>
-
-  <script>
-    // JS
+  <div id="successMessage" style="color: green;"></div>
+  <div id="errorMessage" style="color: red;"></div>
+  <script src="/admin.js">
```

Comment:
Since this table can currently be quite long; may be better to put these error/success messages above the table instead of at the bottom? They're easy to miss right now.